docs/WritingTests.asciidoc

```diff
@@ -40,8 +40,9 @@ openQA tests need to implement at least the *run* subroutine to
 contain the actual test code and the test needs to be loaded in the distribution's
 main.pm.
 
-The *test_flags* subroutine specifies what happens when the test
-fails.
+The *test_flags* subroutine specifies what should happen when test execution is finished.
```

Comment:
In before it stated "if test fails" which would refer to each failure within a job. Now it states "when test execution is finished" which I consider ambiguous. How about "when the execution of the current test module finishes depending on the result …"